guide/README.md

```diff
@@ -455,6 +455,69 @@ used to bump the `Mi 4` note up by one octave and then to bump the
 | | | For example: Transposing C-D-E-F by 4 (fifth) will give us G-A-B-C
 | | | To transpose an octave: shift by the mode length (7 in major scales) up or down. |
 
+#### <a name="Set Key"></a>3.2.5 Set Key
+
+| Set Key Example |
+| --- |
+| ![set key](./setkey1.svg "set key example") |
+
+
+The *Set key* block is used to set the key and mode, e.g, C Major. 
+It is used to change both the mode and key of the current scale. (The current scale is used to define the mapping of Solfege to notes and also the number of half steps take by the the Scalar step block.
+
+The example above demonstrates traveling up and down the major
```

Comment:
the C major scale...

When starting at C, the pattern of a Major mode steps past sharps and flats, consequently there are no sharps or flats in the example.

Reply:
I am not able to properly understand this comment, Do you want me to change 

'The *Set key* block is used to set the key and mode, e.g, C Major. 
It is used to change both the mode and key of the current scale. (The current scale is used to define the mapping of Solfege to notes and also the number of half steps take by the the *Scalar step* block.)'

or 
 to add 

 When starting at C, the pattern of a Major mode steps past sharps and flats, consequently there are no sharps or flats in the example.

 in line 466

Reply:
both